src/__test__/index.test.ts

```diff
@@ -278,15 +278,19 @@ describe('Client', function() {
       existingPerson = entities.pop()!
     })
     it('should stream responses.', done => {
-      client.listen<Person>(store.id, 'Person', existingPerson.ID, reply => {
-        const entity = reply.entity
+      const close = client.listen<Person>(store.id, 'Person', existingPerson.ID, (reply, err) => {
```

Comment:
Changed this to expect a reply _and_ optional error for the callback. This means the caller can properly call close on error (as done in this test). Hopefully this will reduce possible open connections in the future?